rpc/bench_test.go

```diff
@@ -2,18 +2,64 @@ package rpc_test
 
 import (
 	"context"
+	"net"
 	"testing"
 
 	"capnproto.org/go/capnp/v3"
+	"capnproto.org/go/capnp/v3/flowcontrol"
 	"capnproto.org/go/capnp/v3/rpc"
 	testcp "capnproto.org/go/capnp/v3/rpc/internal/testcapnp"
-	"capnproto.org/go/capnp/v3/rpc/transport"
+	"capnproto.org/go/capnp/v3/std/capnp/stream"
 )
 
+func BenchmarkStreaming(b *testing.B) {
+	ctx := context.Background()
+	p1, p2 := net.Pipe()
+	srv := testcp.StreamTest_ServerToClient(nullStream{})
+	conn1 := rpc.NewConn(rpc.NewStreamTransport(p1), &rpc.Options{
+		BootstrapClient: capnp.Client(srv),
+	})
+	defer conn1.Close()
+	conn2 := rpc.NewConn(rpc.NewStreamTransport(p2), nil)
+	defer conn2.Close()
+	bootstrap := testcp.StreamTest(conn2.Bootstrap(ctx))
+	defer bootstrap.Release()
+	var (
+		futures      []stream.StreamResult_Future
+		releaseFuncs []capnp.ReleaseFunc
+	)
+	bootstrap.SetFlowLimiter(flowcontrol.NewFixedLimiter(1 << 9))
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		for j := 0; j < 1<<16; j++ {
```

Comment:
Just in case you haven't already checked, `go test -bench` usually does a pretty good job at picking a large enough `b.N`.  Perhaps we don't need the nested `for` loop?

Reply:
The thing is, b.N looks for the smallest value that gives consistent measurement. here that's not actually what we want -- we need it to send enough messages for re-use to matter, whereas if we just go off b.N, we may not really be using the pool.

Reply:
Makes sense -- just double-checking 👍